### DISCUSSION TITLE
How to display the pixel coordinate of the image of the bounding box

### DISCUSSION THREAD

**Discussion by JunseoKim19** [roboflow/supervision#39] [Q&A]
![image](https://user-images.githubusercontent.com/119910858/224218083-88faeea7-8bea-4ba8-a66b-822406a3fb28.png)

I am using Yolov8 model.
I generated the box using the boxannotator and I want to see the coordinate of the object within the frame. (in x1,y1,x2,y2 form)

I believe it has something to do with get_anchor_coordinate but I just couldn't figure out.
The frame size is 1280 x 720

Thank you

**Comment by SkalskiP** [comment]
Hi @JunseoKim19 👋🏻! `detections.xyxy` will give you `np.ndarray` where every row is a single box in `[x1, y1, x2, y2]` format.
